src/Diagrams/Angle.hs

```diff
@@ -35,18 +35,19 @@ module Diagrams.Angle
        , HasPhi(..)
        ) where
 
-import           Control.Applicative
-import           Control.Lens        (Iso', Lens', iso, review, (^.), over)
-import           Data.Monoid         hiding ((<>))
-import           Data.Fixed
-import           Data.Semigroup
+import Control.Applicative
+import Control.Lens (Iso', Lens', iso, review, (^.), over)
+import Data.Fixed
+import Data.Monoid hiding ((<>))
+import Data.Semigroup
+import Prelude
```

Comment:
What's with all the import reformatting?  I thought we had standardized on the output of `stylish-haskell`.